package javacore.O_exception.runetime.test;

public class RunTimeExceptionTest01 {

	public static void main(String[] args) {
		//Object object = null;
		//System.out.printf("",object.hashCode());
		int [] num = {1,2};
		System.out.print(num[3]);
	}

}
